spec.md

```diff
@@ -0,0 +1,80 @@
+# Gazetteer specification
+
+A gazetteer is a [GeoJSON](https://tools.ietf.org/html/rfc7946) FeatureCollection object. Each entry (aka "place") is represented by a Feature object:
+
+```
+{
+	"type": "FeatureCollection",
```

Comment:
Should we add a top-level property for the name of the gazetteer? What about for the name of the data source this gazetteer references?

Reply:
We can @nickidlugash right now when you upload a gazetteer, we hold onto the filename. We can add that as the default value on download as well. Do you think we still want a top level property for the name or the gazetteer? For the record I think this would be easy to add in later since we're not adding multi-gazetteer functionality yet. @dasulit do you agree?